deploy/role.yaml

```diff
@@ -51,13 +51,19 @@ metadata:
   name: {{template "handlerPrefix" .}}nmstate-handler
   namespace: {{ .HandlerNamespace }}
 rules:
-- apiGroups:
-  - certificates.k8s.io
+  # Istiod and bootstrap.
```

Comment:
Why do we need istio and what is bootstrap?

Reply:
Copy paste from issue fixd at isstio related to k8s 1.18 also we are facing the same at ocp 4.4 but it only happends if we create csrs ourselves, if we go to openshift solution we don't care for this.

Reply:
If we know what it stands for, we can keep the comment. It is just a copy-paste with no descriptive value for us, we should drop it.